src/CardStatistics.js

```diff
@@ -40,17 +40,18 @@ class CardStatistics extends React.Component {
 		return (
 			<div className="statistics">
 				<div className="border-bottom flex-row">
-					<div style={{ flex: '0 0 35%' }}>
-						<div className="percentage purple">
-							{rate}%
-						</div>
-					</div>
-					<div 
-						className="five-year font-size-3"
-						style={{ display: 'flex' }}
-					>
-							5 year survival rate
-					</div>
+					
+					<div className="statistic-line"> 
+								<div className="percentage bg-green">
+									{rate}%
+								</div>
+							</div>
+							<div 
+								className="survival-line"
+								style={{  }}
```

Comment:
Remove `style={{ }}`
Lines 50 through 52 can be combined into `<div className="survival-line">`